### PR TITLE
Fix&Test to Format::_from_xml() and Better suport to customize links in Pagination class

### PR DESCRIPTION
**Pull request by kriansa**
Well, I'm still newbie in git, so forgive-me if I'm doing something wrong.

I made some fixes on my last pull request, but I screw everything up with a bunch of unwanted commits. Now I'm doing as requested by @philsturgeon.

The first is a real fix to issue #432. The actual function Format::_from_xml() convert the XML and its children as SimpleXMLElement objects, instead arrays (as expected).

In my test, I use to_php() to show you clearly that it's really unexpected.

And the commit b41b62f90a5a9fceaf80fe719557d21461d577b2 adds a better support to customize the unclickable links << Previous and Next>> when they're inactive(when you're in first or last page), so we can set a custom CSS class on them.

Thanks.

### CODE REVIEW

File: classes/pagination.php [hunk 1]
```diff
@@ -226,7 +228,7 @@ class Pagination {
 
 		if (static::$current_page == static::$total_pages)
 		{
-			return $value.static::$template['next_mark'];
```
Comment (philsturgeon) [inline classes/pagination.php:229]: Can you do something about these changes showing up?

Comment (philsturgeon) [inline classes/pagination.php:229]: Oh sorry I hadn't noticed that this was an intended change. Generally we ask people to make sure only one "change" happens per pull request.

Comment (kriansa) [inline classes/pagination.php:229]: I see. Is there any way to accept just one diff of a pull request or should I make another one, or two?

Comment (philsturgeon) [inline classes/pagination.php:229]: Thanks for making the XML file in a new pull request, this has been merged to develop. Please make the validation pull request fresh too.

**Comment by kriansa** [comment]
Okay... Now I get it!

https://github.com/fuel/core/pull/578

and

https://github.com/fuel/core/pull/577

Thanks!

**Pull request closed without merging**